src/components/Audioplayer/index.js

```diff
@@ -231,6 +280,24 @@ export default class Audioplayer extends Component {
     );
   }
 
+  renderScrollButton() {
+    const { shouldScroll } = this.props;
+
+    return (
+      <Col xs={2} className="text-center pull-right">
+        <input type="checkbox" id="scroll" className={style.checkbox} />
+        <label
+          htmlFor="scroll"
+          className={`pointer ${style.buttons} ${shouldScroll ? style.scroll : ''}`}
+          onClick={this.scroll.bind(this)}
+        >
+          <i className="ss-icon ss-openbook" />
```

Comment:
I feel like users might not understand what this means. We should probably use another icon or text. See https://github.com/quran/quran.com-frontend/blob/master/static/fonts/ss-standard/documentation.html